types/math-toolbox/index.d.ts

```diff
@@ -0,0 +1,582 @@
+/**
+ * Constrain a value to lie between two further values.
+ *
+ * @param  {number} min - Lower end of the range into which to constrain v.
```

Comment:
JSDoc type hints like `{number}` aren't needed in TypeScript files

Reply:
oh! sorry
The file was autogenerated from jsdoc comments I just added them here 
Should I remove them?

Reply:
I'd recommend it, since it can be confusing with Typescript's types already there

Reply:
got it thanks!
will fix it tomorrow gotta sleep now 😴